grails-app/controllers/org/pih/warehouse/product/ProductController.groovy

```diff
@@ -1022,6 +1016,11 @@ class ProductController {
     }
 
 
+    def synonyms = {
```

Comment:
Is this used?

Reply:
Good catch. I changed a bunch of these from using controller actions to just using renderTemplate since they all basically did the same thing. I'll try to remove the dead code when I get a chance.